src/options.cpp

```diff
@@ -1458,7 +1463,6 @@ void options_manager::add_options_interface()
 
     add( "MORALE_STYLE", "interface", translate_marker( "Morale style" ),
          translate_marker( "Morale display style in sidebar." ),
-         //~ aim bar style - bars or numbers
```

Comment:
Why did you remove this translator comment?

Reply:
Thought it was just a misplaced copy paste- is morale display actually the same as the aiming style option?

Reply:
Oh, didn't notice that was a copypaste. Yeah, you're right. You could alter the comment to correctly represent what should be translated, or remove the comment entirely, on your discretion.